### ISSUE TITLE
Specifying npm_dependencies does not seem to work correctly

### ISSUE THREAD

**Issue by brandonwkipp** [browserstack/browserstack-cypress-cli#80]
Hey there,

I'm trying to evaluate `Browserstack` and I can't seem to get my config to play nice with this tool. I'm trying to use the `cypress-axe` dependency by including it in my config like this:

```
{
    "auth": {
        "username": "SOME_USER",
        "access_key": "SOME_STRING"
    },
    "browsers": [
        {
            "browser": "chrome",
            "os": "OS X Catalina",
            "versions": [
                "latest"
            ]
        }
    ],
    "run_settings": {
        "cypress_config_file": "./cypress.json",
        "cypress_proj_dir": "./",
        "project_name": "sandbox",
        "build_name": "cypress-test",
        "exclude": [],
        "npm_dependencies": {
            "axe-core": "^4.0.2",
            "cypress": "^3.8.2",
            "cypress-axe": "^0.8.1"
        },
        "package_config_options": {},
        "parallels": "5",
        "specs": [
            "./cypress/integration/app-nav.js"
        ]
    },
    "connection_settings": {
        "local": false,
        "local_identifier": null
    },
    "disable_usage_reporting": false
}
```

When I run `browserstack-cypress run` and check the Dashboard, the test fails with this output:
```
Oops...we found an error preparing this test file:

  /e2e/cypress/support/index.js

The error was:

Error: Can't walk dependency graph: Cannot find module 'cypress-axe' from '/e2e/cypress/support'
    required by /e2e/cypress/support/index.js

This occurred while Cypress was compiling and bundling your test code. This is usually caused by:

- A missing file or dependency
- A syntax error in the file or one of its dependencies

Fix the error in your code and re-run your tests.
```

I've read the docs and I don't think I'm missing anything, but I'm almost out of free minutes trying to evaluate this product. Can anyone point me in the right direction?

**Comment by Namrata-m13** [comment]
@brandonwkipp to debug this further, could you please share the respective build id or session id as well as other required details with the BrowserStack support team over support@browserstack.com?

**Comment by webunderconstruction** [comment]
@Namrata-m13 getting the same issue, error popped up 4 days ago, cypress config hasn't changed

**Comment by brandonwkipp** [comment]
It looks like this issue is not caused by Browserstack and is instead caused by [a regression](https://github.com/component-driven/cypress-axe/issues/70) within the `cypress-axe` library. The regression has since been fixed and upgrading to the patched version (0.9.1) fixes the issue.
